tests/publiccloud/prepare_tools.pm

```diff
@@ -93,7 +93,7 @@ sub run {
     record_info('img-proof', script_output('img-proof --version'));
 
     # Install Terraform from repo
-    zypper_call('ar https://download.opensuse.org/repositories/systemsmanagement:/terraform/SLE_15/systemsmanagement:terraform.repo');
+    zypper_call('ar https://download.opensuse.org/repositories/systemsmanagement:/terraform/SLE_15_SP1/systemsmanagement:terraform.repo');
```

Comment:
Because you have generalized the autoyast profile. Maybe we could do it here to?
```
(my $v = uc(join('_', get_var('DISTRI'), get_var('VERSION'))) =~ s/-/_/g;
```
I thought there was a function which is doing it, but I didn't found it.

Reply:
We could, but it would only help if we upgrade to SP2.
It's only available on SP1 and SP2 in 
https://download.opensuse.org/repositories/systemsmanagement:/terraform/
So, some more options would fail (e.g. SLE12, SLE15).

Reply:
taking into account how much time and work it took just migrate from SP0 to SP1 I don't see the point . We have too many things moving here . So I would say that the chance that you will change version from SP1 to SP2 and it will "just works" are close to 0 :)

Reply:
... maybe I should replace `{{VERSION}}` back to `15.1` than ?

Reply:
Also fine for me. But lets keep `{{VERSION}}`.